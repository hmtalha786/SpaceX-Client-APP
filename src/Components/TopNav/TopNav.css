.top-nav{
    text-align: center;
    background-color: rgb(255, 255, 255);
    margin: 0;
}
.spacex-logo{
    width: 150px;
    margin: 8px auto;
}