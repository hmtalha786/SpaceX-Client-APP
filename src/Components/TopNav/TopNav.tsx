import React from 'react'
import './TopNav.css'

const TopNav = () => {
    return (
        <div className="top-nav">
            <img className="spacex-logo" src="https://upload.wikimedia.org/wikipedia/commons/9/96/SpaceX_Logo_Black.png" alt="spacex-logo" />
        </div>
    )
}

export default TopNav
