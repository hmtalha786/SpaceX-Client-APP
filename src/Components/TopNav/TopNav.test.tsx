import React from 'react'
import {shallow} from 'enzyme'
import TopNav from './TopNav';

describe('TopNav test case', () => {
    let container : any;

    beforeEach(() => container = shallow(<TopNav />))

    it('renders the TopNav' , () => {
        expect(container).toMatchSnapshot();
    })
})
