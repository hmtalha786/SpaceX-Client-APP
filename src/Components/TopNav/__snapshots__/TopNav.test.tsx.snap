// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav test case renders the TopNav 1`] = `
<div
  className="top-nav"
>
  <img
    alt="spacex-logo"
    className="spacex-logo"
    src="https://upload.wikimedia.org/wikipedia/commons/9/96/SpaceX_Logo_Black.png"
  />
</div>
`;
